update audio docs (#1455)
cocos-creator/creator-docs: en/audio-system/audiosource.md

# AudioSource Component

![](audio/audiosource.png)

## Properties

|Properties          | Function Explanation                                                    |
|:--                 | :--                                                                     |

> |Clip                | Audio resource object to play                                           |

Reviewer: ```suggestion
|Clip                | The audio resource object to be played                                           |

```

> |Loop                | Whether it is loop                                                      |

Reviewer: ```suggestion
|Loop                | Whether to loop                                                      |

```

|PlayOnAwake         | Whether to play audio automatically after the component is activated    |

> |Volume              | Audio volume, the range is between 0~1                                  |

Reviewer: ```suggestion
|Volume              | Audio volume, ranging from 0 to 1                                  |

```

> More audio interface to the script interface [AudioSource API](../../../api/en/classes/component_audio.audiosource.html).  

Reviewer: ```suggestion
For more script interfaces for sound interfaces, please refer to [AudioSource API](../../../api/en/classes/component_audio.audiosource.html).  

```

> For specific playback controls, refer to the document [Audio System Overview](./overview.md).

Reviewer: ```suggestion
For specific playback controls, please refer to [Audio System Overview](./overview.md).

```
